packages/core/src/cli/commands.ts

```diff
@@ -82,17 +82,19 @@ export function setupCommands(): void {
   applyCommonOptions(cli);
 
   cli
-    .command('[...filters]', 'run Rstest')
+    .command('[...filters]', 'run tests in watch mode')
     .action(async (filters: string[], options: CommonOptions) => {
       try {
         const { config } = await initCli(options);
         const { createRstest } = await import('../core');
-        if (isCI) {
```

Comment:
Do we need to auto turn off `watch` mode in CI?

Reply:
`rstest watch` will always run watch. `rstest` will enter the watch mode in development environment and run mode in CI automatically.

Reply:
Use a build phase define to escape as a workaround.